vigor/static/css/style.css

```diff
@@ -0,0 +1,73 @@
+@import "../bower_components/material-typography/css/typography.css";
```

Comment:
wait. you are importing it in the CSS... Maybe Bower hasn't updated the versions yet, and it's still on 1.0.7 before all those typography updates.